src/explore-education-statistics-frontend/src/modules/table-tool/TableToolPage.tsx

```diff
@@ -103,90 +103,99 @@ class TableToolPage extends Component<Props, State> {
-                    </Link>
+                    {permalinkId ? (
+                      <>
+                        <div>Generated permanent link:</div>
+                        <LinkContainer
+                          url={`${window.location.host}/data-tables/permalink/${permalinkId}`}
+                        />
+                        <div>
+                          <a
+                            className="govuk-link"
+                            href={`/data-tables/permalink/${permalinkId}`}
+                            title="View created table permalink"
+                            target="_blank"
+                            rel="noopener noreferrer"
+                          >
+                            View permanent link
+                          </a>
+                        </div>
+                      </>
+                    ) : (
+                      <>
+                        {permalinkLoading ? (
+                          <>
+                            Generating permanent link
+                            <LoadingSpinner inline size={19} />
+                          </>
+                        ) : (
+                          <ButtonText
+                            disabled={permalinkLoading}
+                            onClick={() =>
+                              this.handlePermalinkClick({ query, tableHeaders })
+                            }
+                          >
+                            Generate permanent link
+                          </ButtonText>
+                        )}
+                      </>
+                    )}
                   </li>
                 </ul>
-              )}
-              <p className="govuk-body">
-                If you have a question about the data or methods used to create
-                this table contact the named statistician via the relevant
-                release page.
-              </p>
-            </>
-          )}
+
+                <h3>Additional options</h3>
+
+                {publication && (
+                  <ul className="govuk-list">
+                    <li>
+                      <Link
+                        as={`/find-statistics/${publication.slug}`}
+                        to={`/find-statistics/publication?publication=${publication.slug}`}
+                      >
+                        Go to publication
+                      </Link>
+                    </li>
+                    <li>
+                      <DownloadCsvButton
+                        publicationSlug={publication.slug}
+                        fullTable={createdTable}
+                      />
+                    </li>
+
+                    <li>
+                      <a href="#api">Access developer API</a>
+                    </li>
+                    <li>
+                      <Link
+                        as={`/methodology/${publication.slug}`}
+                        to={`/methodology/methodology?publication=${publication.slug}`}
```

Comment:
Is this the correct URL?

/methodology/methodology

Reply:
I haven't actually changed the url, it's probably just been reformatted. This url was committed by Lewis on 7/8/19 ... I'm not sure if it's correct or not.